src/command_modules/azure-cli-cdn/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.1.2
++++++
+* `cdn endpoint create`: POSSIBLE BREAKING CHANGE. Default query string caching behaviour no longer defaults to "IgnoreQueryString". It is now set by the service.
```

Comment:
There are no "possible" breaking changes. Just announce it as a breaking change (first) and the version number should be 0.2.0